docs/quick-start.md

```diff
@@ -33,7 +34,7 @@ Otherwise you can build and start your app directly:
 
 ```bash
 npm run build
-npm run start --env local
+GASKET_ENV=local npm run start
```

Comment:
```suggestion
npm run local
```

Reply:
I'm not sure what the change is here. I have the local command in the section above. Do you want me to remove the entire seciton?

Reply:
My bad... 

So, to be concise, leave out the GASKET_ENV in the example.
```
npm run build
npm run start
```

local is the default env now